k8s/aat/common/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -15,12 +22,42 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: bulk-scan-orchestrator
-    version: 0.2.9
+    version: 0.2.10
```

Comment:
Would be nice if version bump is separate pr as if something goes wrong then easy to revert

Reply:
Good thinking. Done in #3534